docs/docs.md

```diff
@@ -4932,6 +4935,37 @@ Configuration for Google Chat notifications.
 
 
 
+<a name="notification/microsoft_teams.proto"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## notification/microsoft_teams.proto
+
+
+
+<a name="proto.notification.MicrosoftTeams"></a>
+
+### MicrosoftTeams
+Configuration for Microsoft Teams notifications.
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| enabled | [google.protobuf.BoolValue](#google.protobuf.BoolValue) |  | Whether Microsoft Teams notifications are enabled. |
+
+
+
+
+
+ 
+
+ 
+
+ 
+
+ 
+
+
+
```

Comment:
[note]
Theses are auto generated by the `kleat-protobuilder`. I ran `make proto`.